lib/pages/discover.dart

```diff
@@ -42,6 +44,8 @@ class _DiscoverDevicesState extends State<DiscoverDevices> {
     this.scan();
   }
 
+  RefreshController _refreshController = RefreshController(initialRefresh: false);
```

Comment:
We are calling the `scan()` function in `initState()`. Now that we have Refresher, we can get rid of `scan()` in `initState()` and toggle `initialRefresh` to `true`.

Reply:
@sanket143 ok got it, but here should I only remove the call to `scan()` function in `initState()` or the complete `initState()` function itself?

Reply:
Yeah, you can remove `initState()`.